python/lsst/daf/butler/core/quantum.py

```diff
@@ -99,39 +84,26 @@ class Quantum:
     def __init__(self, *, taskName: Optional[str] = None,
                  taskClass: Optional[Type] = None,
                  dataId: Optional[DataCoordinate] = None,
-                 run: Optional[str] = None,
                  initInputs: Optional[Union[Mapping[DatasetType, DatasetRef], Iterable[DatasetRef]]] = None,
                  predictedInputs: Optional[Mapping[DatasetType, List[DatasetRef]]] = None,
```

Comment:
With dropping `actualInputs` would it make sense to call it `inputs` instead of `predictedInputs`?